xml/containers-buildah-overview.xml

```diff
@@ -0,0 +1,211 @@
+   <para>
+    Delete a working container or an image and rename a local container.
+   </para>
+  </listitem>
+ </itemizedlist>
+ <para>
+  Compared to &deng;, &buildah; has several advantages.
+ </para>
+ <itemizedlist>
+  <listitem>
+   <para>
+    The tool makes it possible to mount a working container's filesystem, so it
+    becomes accessible by the host.
+   </para>
+  </listitem>
+  <listitem>
+   <para>
+    The process of building container images using &buildah; can be automated
+    via scripts by using &buildah;'s subcommands instead of a Containerfile or
+    Dockerfile.
+   </para>
+  </listitem>
+  <listitem>
+   <para>
+    Similar to &podman;, &buildah; does not require a daemon to run and can be
+    used by unprivileged users.
+   </para>
+  </listitem>
+  <listitem>
+   <para>
+    It is possible to build images inside a container without mounting the
+    &docker; socket, which improves security.
+   </para>
+  </listitem>
+ </itemizedlist>
+ <sect1 xml:id="cha-podman-buildah">
+  <title>&podman; and &buildah;</title>
+
+  <para>
+   Both &podman; and &buildah; can be used to build container images. While
+   &podman; makes it possible to build images using Dockerfiles, &buildah;
+   offers an expanded range of image building options and capabilities.
+  </para>
+ </sect1>
+ <sect1 xml:id="cha-buildah-install">
+  <title>&buildah; Installation</title>
+
+  <para>
+   To install &buildah;, run the <command>sudo zypper in buildah</command>. Run
+   then <command>buildah --version</command> to check whether &buildah; has
+   been installed successfully.
+  </para>
+
+  <para>
+   If you already have &podman; installed and set up for use in the rootless
+   mode, &buildah; can be used in an unprivileged environment without any
+   further configuration. If you have to enable the rootless mode for
+   &buildah;, run the following command:
+  </para>
+
+<screen>&prompt.user;sudo usermod --add-subuids 200000-201000 --add-subgids
+   200000-201000 $USER</screen>
+
+  <para>
+   This command enables the rootless mode for the current user. After running
+   the command, log out and log in again to enable the changes.
+  </para>
+
+  <para>
+   The command above defines a range of local uids to which the uids allocated
+   to users inside the container are mapped on the host. Note that the ranges
+   defined for different users must not overlap. It is also important that the
+   ranges do not reuse the uid of an existing local user or group. By default,
+   adding a user with the <command>useradd</command> on &slsa; &product-ga;,
+   automatically allocates subuid and subgid ranges.
+  </para>
+
+  <note>
+   <title>&buildah; in the rootless mode</title>
+   <para>
+    In the rootless mode, &buildah; commands must be executed in a modified
+    user namespace of the user. To enter the user namespace, use the
+    <command>buildah unshare</command> command. Otherwise, the <command>buildah
+    mount</command> command will fail.
```

Comment:
```suggestion
    In its rootless mode, &buildah; commands must be executed in a modified
    user namespace of the user. To enter this user namespace, run the command
    <command>buildah unshare</command>. Otherwise, the <command>buildah
    mount</command> command will fail.
```

Reply:
I have tried to simplify this a little, including removing one of many occurrences of forms of the word "use". However, I think the phrase:

>  a modified user namespace of the user

... is confusing. "User namespace of the user"? Of **what** user?